changelog/fragments/k8s-1.24-bump.yaml

```diff
@@ -0,0 +1,34 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Bump dependencies to support Kubernetes 1.24. Projects scaffolded with the go/v3 plugin now support Go 1.18.
+      Dependency bumps that occurred are as follows:
+      
+      github.com/operator-framework/api --> v0.15.0
```

Comment:
we need add the changelog here, see; 

PS: these changes also goes to the changelogs:

:bug: fix channel naming validation by [@camilamacedo86](https://github.com/camilamacedo86) in [#231](https://github.com/operator-framework/api/pull/231)
Add CRD description check validation by [@ryantking](https://github.com/ryantking) in [#234](https://github.com/operator-framework/api/pull/234)
:sparkles: (GoodPracticesValidator) : Add a new check to warn authors when permissions to create CRDs are found by [@camilamacedo86](https://github.com/camilamacedo86) in [#241](https://github.com/operator-framework/api/pull/241)

[Camila Macedo](https://app.slack.com/team/UM6JTJBGV):slightly_smiling_face:  [1 day ago](https://coreos.slack.com/archives/GCV27CWNP/p1654696520009529?thread_ts=1654696368.895799&cid=GCV27CWNP)
Since change/additions valid for those who use SDK bundle validate command

i.e.: ( from previous bumps ) : see: https://github.com/operator-framework/operator-sdk/blob/c9c61b6921b29d731e64cd3cc33d268215fb3b25/changelog/generated/v1.18.0.md#L6

Reply:
we do not tell the end-user that we upgrade operator-framework api we just add here the changes that came with and are relevant to them  (like the i.e ^)

Reply:
Okay, I can change that.

Reply:
Updated it in this commit: https://github.com/operator-framework/operator-sdk/pull/5843/commits/b7b0924a3a2aef6416899e81f51d754e32f0ce2b